diff -Nur old/pcmm/pcmm.desktop new/pcmm/pcmm.desktop
--- old/pcmm/pcmm.desktop	2002-10-17 20:46:19.000000000 +0000
+++ new/pcmm/pcmm.desktop	2004-02-12 11:17:05.000000000 +0000
@@ -1,12 +1,13 @@
 # KDE Config File  
 [Desktop Entry]
+Name=Portos Commander
+Comment=Portos Commander file manager
+Comment[pl]=Zarządca plików Portos Commander
 Type=Application
 Exec=pcmm -caption "%c" %i %m  
 Icon=pcmm.png
 MiniIcon=pcmm.png
-DocPath=pcmm/index.html
-Comment=
-Comment[de]=
-Terminal=0
-Name=Pcmm
-Name[de]=Pcmm
+Terminal=false
+Categories=Qt;KDE;Utility;FileManager;
+Encoding=UTF-8
+# vi: encoding=utf-8
